test/Npgsql.Tests/BatchTests.cs

```diff
@@ -136,6 +136,52 @@ public async Task RecordsAffected_and_Rows()
             Assert.That(command.Rows, Is.EqualTo(2));
         }
 
+        [Test]
+        public async Task RecordsAffected_and_Rows15()
```

Comment:
Can you please also add a statement in ReaderTest.RecordsAffected (to cover non-batching usage?)

Reply:
Thanks done